docs/03-contributing/01-getting_started.md

```diff
@@ -159,10 +180,7 @@ If you have messenger jobs configured, be sure to stop them:
 - Docker: `docker compose stop messenger`
 - Bare Metal: `supervisorctl stop messenger:*`
 
-If you are using the Docker setup and want to load the fixture.
-
-1. Go to the `docker` directory.
-2. Execute: `docker compose exec php bin/console doctrine:fixtures:load --append --no-debug`
+If you are using the Docker setup and want to load the fixture, execute: `docker compose exec php bin/console doctrine:fixtures:load --append --no-debug`
```

Comment:
```suggestion
If you are using the Docker setup and want to load the fixture, execute: 

````
docker compose exec php bin/console doctrine:fixtures:load --append --no-debug
```
```

Lets move it to a next line with a bigger code block.